app/src/main/res/layout/fragment_account_setting.xml

```diff
@@ -23,6 +23,7 @@
             android:elevation="1dp"
             android:background="@drawable/round_outline"
             android:src="@drawable/ic_avatar_placeholder"
+            android:scaleType="centerCrop"
```

Comment:
ooOOoo

Reply:
if this is a better solution than using the custom Picasso transformation, let's see if we can do it in both places (IIRC in the drawer as well?)

Reply:
doneeee

Reply:
We can't remove the `.transform(CropCircleTransformation())` in the nav drawer, possibly because it's nested p deep in the Navigation View. But I made sure that the rest of the resources are identical in both places.